src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({ name }) {
+  return (
+    <div>
```

Comment:
This is a `button component`, you should use a `button` tag instead of a `div` tag

Reply:
Makes sense. Made the change.